src/Sfa.Tl.Matching.Data/SearchProviders/DummySearchProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Sfa.Tl.Matching.Data.Interfaces;
+using Sfa.Tl.Matching.Domain.Models;
+
+namespace Sfa.Tl.Matching.Data.SearchProviders
+{
+    public class DummySearchProvider : ISearchProvider
+    {
+        private readonly ILogger<DummySearchProvider> _logger;
+
+        public DummySearchProvider(ILogger<DummySearchProvider> logger)
+        {
+            _logger = logger;
+        }
+
+        public Task<IEnumerable<ProviderVenueSearchResult>> SearchProvidersByPostcodeProximity(string postcode, int searchRadius, int routeId)
```

Comment:
Just rename this to ProviderVenueSearchResultDto and you can use that all the way up to web